services/rfq/relayer/service/statushandler.go

```diff
@@ -81,6 +98,10 @@ func (r *Relayer) requestToHandler(ctx context.Context, req reldb.QuoteRequest)
 		apiClient:           r.apiClient,
 		mutexMiddlewareFunc: r.mutexMiddleware,
 		handlerMtx:          r.handlerMtx,
+		volumeLimit:         r.cfg.GetVolumeLimit(int(origin.ChainID), req.Transaction.OriginToken),
```

Comment:
this doesn't appear to be used anywhere 
<img width="298" alt="image" src="https://github.com/user-attachments/assets/61711de1-a77d-44bd-a7ad-2e4875d45df7">